man/systemd.netdev.xml

```diff
@@ -185,6 +185,13 @@
 
           <row><entry><varname>bareudp</varname></entry>
           <entry> Bare UDP tunnels provide a generic L3 encapsulation support for tunnelling different L3 protocols like MPLS, IP etc. inside of an UDP tunnel.</entry></row>
+
+          <row><entry><varname>batadv</varname></entry>
+          <entry> (<ulink url="https://www.open-mesh.org/projects/open-mesh/wiki">B.A.T.M.A.N. Advanced</ulink>)
+           is a new approach to wireless networking which does no longer operate on the IP basis. Unlike the
+           batman daemon, which exchanges information using UDP packets and sets routing tables,
+           batman-advanced operates on ISO/OSI Layer 2 only and uses and routes (or better: bridges) Ethernet
+           Frames. It emulates a virtual network switch of all nodes participating.</entry></row>
```

Comment:
this reads a bit too much like an advertisement. "new approach" tends to age quickly. Please shorten. Also, please drop the references to the batman daemon, i doubt people who read this might know it. just say what it is in technical, "timeless" words what it is.

Reply:
Just copied the original text of the manpage. But you are right.  It's not timeless, will get to the BATMAN developers to rephrase :).